src/content-handlers/iiif/IIIFContentHandler.ts

```diff
@@ -397,6 +380,27 @@ export default class IIIFContentHandler
 
       data.config = await that.configure(config);
 
+      // if using uv-av-extension and there is no structure,
+      // or the preferMediaElementExtension config is set
+      // fall back to uv-mediaelement-extension
+      const hasRanges: boolean = helper.getRanges().length > 0;
+
+      if (
+        (extension!.type === Extension.AV && !hasRanges) ||
+        (extension!.type === Extension.AV &&
+          data.config.options.preferMediaElementExtension)
```

Comment:
This is equivalent but feels slightly simpler:

```suggestion
        (extension!.type === Extension.AV && (!hasRanges || data.config.options.preferMediaElementExtension))
```

I'm fine with leaving it as-is if you prefer it that way, though; I realize it's a matter of personal preference!

Reply:
yep, have pushed an update